handlers/high-charts/src/utils.ts

```diff
@@ -88,3 +88,35 @@ export async function sendOSC(jsonFile: string, outFile: string, server: string,
         })
     ]);
 }
+
+/**
+ * The function will return the string representing the graph title and axes information
+ * @param highChartsData 
+ * @returns graphInfo string
+ */
+export function getGraphInfo(highChartsData: any): string{
+    const chartsData = JSON.parse(JSON.stringify(highChartsData));
+    const title = chartsData.title || chartsData.series[0].name || 'Untitled Chart';
+    const xAxis = chartsData.axes.find((axes: { axis: string; })=>axes.axis == "xAxis");
+    const yAxis = chartsData.axes.find((axes: { axis: string; })=>axes.axis == "yAxis");
+    let xStart = xAxis.dataMin;
+    let xEnd = xAxis.dataMax;
+    let yStart = yAxis.dataMin;
+    let yEnd = yAxis.dataMax;
+    const monthNames = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"];
```

Comment:
I recommend using `Intl.DateTimeFormat` as shown on the [MDN Date.getMonth page](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getMonth).
This will make it easier to add French (and other) language support later. For example:
```
> new Intl.DateTimeFormat("en-US", { month: 'long' }).format(Date.now())
'June'
> new Intl.DateTimeFormat("fr-CA", { month: 'long' }).format(Date.now())
'juin'
```